index.html

```diff
@@ -705,3 +706,33 @@ <h4 id="sql-linux">Linux</h4>
   </ol>
 </div>
 -->
+
+<!--
+  DOWNLOADS
+
+  Links to learner downloads (e.g. datasets needed by lessons).
+  Delete/modify as needed to match the lessons you are covering.
+-->
+
+<h2 id="downloads">Downloads</h2>
+
+<p>
+  The following downloads and data-sets will be used during the
+  workshop.  Please download and save them in a location you can find
+  for the workshop (for example, your desktop or your downloads
+  directory).
+</p>
+
+<ul class="downloads">
+  <!-- for swcarpentry/shell-novice -->
+  <li><a href="https://swcarpentry.github.io/shell-novice/shell-novice-data.zip"><code>shell-novice-data.zip</code></a></li>
+  <!-- for swcarpentry/python-novice-inflammation -->
+  <li><a href="https://swcarpentry.github.io/python-novice-inflammation/python-novice-inflammation-data.zip"><code>python-novice-inflammation-data.zip</code></a></li>
+  <!-- for swcarpentry/r-novice-inflammation -->
+<!--
+  <li><a href="https://swcarpentry.github.io/r-novice-inflammation/r-novice-inflammation-data.zip"><code>r-novice-inflammation-data.zip</code></a></li>
+-->
+  <!-- for swcarpentry/sql-novice-survey -->
+  <li><a href="http://files.software-carpentry.org/survey.db"><code>survey.db</code></a></li>
+  <li><a href="http://files.software-carpentry.org/survey.sql"><code>survey.sql</code></a></li>
```

Comment:
On Wed, Mar 23, 2016 at 08:37:54PM -0700, Elliott Sales de Andrade wrote:
> > +  <!-- for swcarpentry/sql-novice-survey -->
> > +  <li><a href="http://files.software-carpentry.org/survey.db"><code>survey.db</code></a></li>
> > +  <li><a href="http://files.software-carpentry.org/survey.sql"><code>survey.sql</code></a></li>
> 
> This can (and should) use https: links.

That didn't work when I wrote the commit (see the commit message for
6b9f36c47 and swcarpentry/site#509), but it was fixed in on 2016-01-11
[1].  I've updated this PR (and rebased to resolve trivial conflicts
with #303) in 4fb0eb1→7f67d1e.

[1]: https://github.com/swcarpentry/site/issues/509#issuecomment-170674095